### PR TITLE
KSE3-522 Charts

### PR DESCRIPTION
**Pull request by mariokleen**
- Line and Pie chart

chart

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/Line/Line.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    },
+    lineColor: 'var(--grey_mid_0x)',
+    tickColor: 'var(--grey_mid_0x)',
+    title: {
+      text: '',
+      style: {
+        fontSize: 'var(--tx-2XS)',
+        fontWeight: 'light',
+        color: 'var(----primary-color)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Line/Line.tsx:73]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Line/Line.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    },
+    lineColor: 'var(--grey_mid_0x)',
+    tickColor: 'var(--grey_mid_0x)',
+    tickWidth: 1,
+    title: {
+      text: '',
+      style: {
+        fontSize: 'var(--tx-2XS)',
+        fontWeight: 'light',
+        color: 'var(----primary-color)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Line/Line.tsx:52]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Line/Line.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      },
+    }
+  },
+  legend: {
+    enabled: false,
+    backgroundColor: 'var(--transparent)',
+    itemStyle: {
+      fontSize: 'var(--tx-1XS)',
+      fontWeight: 'light',
+      color: 'var(----primary-color)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Line/Line.tsx:90]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Line/Line.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    },
+    itemHoverStyle:{
+      color: 'var(----primary-color-light)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Line/Line.tsx:93]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Pie/Pie.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  },
+  legend: {
+    align: 'right',
+    layout: 'vertical',
+    verticalAlign: 'middle',
+    backgroundColor: 'var(--transparent)',
+    itemStyle: {
+      fontSize: 'var(--tx-S)',
+      fontWeight: 'light',
+      color: 'var(----primary-color)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Pie/Pie.tsx:51]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Pie/Pie.tsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    itemHoverStyle:{
+      color: 'var(----primary-color-light)'
```
Comment (EdisonKleeen) [inline libs/atomic-elements/src/lib/components/Pie/Pie.tsx:54]: should it be (--primary-color)?

File: libs/atomic-elements/src/lib/components/Line/Line.tsx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+export const Line = (props: HighchartsReact.Props) => {
+  const results = props.context.data.results || [];
+  const headers = props.context.data.headers || ['','loading'];
+  const categories = [];
+  const series =  [{
+    name: headers[1].replace(/([A-Z]+)/g, " $1").replace(/ .*/,'').replace(/\b[a-z]/g, (x) => x.toUpperCase()),
+    data: [],
+    tooltip: {
+      valuePrefix: '$',
```
Comment (sebaskleeen) [inline libs/atomic-elements/src/lib/components/Line/Line.tsx:104]: dont know if this is as an example only, but not everything will be expenses right?  `$$$`

Comment (mariokleen) [reply libs/atomic-elements/src/lib/components/Line/Line.tsx:104]: Unfortunately, this is the same as before. However, all this code will eventually go away once KAPI gets updated and sends in the correct data.

**Pull request merged**